src/amo/components/RatingsByStar/styles.scss

```diff
@@ -0,0 +1,60 @@
+@import '~photon-colors/photon-colors';
+@import '~core/css/inc/vars';
+@import '~core/css/inc/mixins';
+
+.RatingsByStar-graph {
+  color: $grey-50;
+  display: grid;
+  font-size: $font-size-default;
+  grid-gap: 12px;
+  grid-template-columns: max-content auto max-content;
+  line-height: normal;
+}
+
+.RatingsByStar-star {
+  align-items: center;
+  display: flex;
+
+  .Icon {
+    @include margin-start(6px);
+  }
+}
+
+.RatingsByStar-barContainer {
+  align-items: center;
+  display: flex;
```

Comment:
I dont know that either style is needed here

Reply:
without this, the bar was not vertically centered. Since the bar has a height of 16px there was some small whitespace but I wonder now if those were due to `line-height` problems that I've since fixed.

Reply:
hmm ok, I didn't see this...

I guess leave if you see something else/it's breaking..

Reply:
Actually, it made me think to force a line-height of `16px` which fixes everything 🌤 I could then remove the vertical centering.